portal-api/src/v1/gef/services/clone-deal.service.ts

```diff
@@ -0,0 +1,170 @@
+import { ObjectId } from 'mongodb';
+import { produce } from 'immer';
+import {
+  AuditDetails,
+  Bank,
+  getCurrentGefDealVersion,
+  PortalSessionUser,
+  GefDeal,
+  MONGO_DB_COLLECTIONS,
+  InvalidDealIdError,
+  InvalidParameterError,
+  DealNotFoundError,
+  AnyObject,
+} from '@ukef/dtfs2-common';
+import { generateAuditDatabaseRecordFromAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { mongoDbClient } from '../../../drivers/db-client';
+import { getLatestEligibilityCriteria } from '../controllers/eligibilityCriteria.controller';
+import { DEAL } from '../../../constants';
+import { cloneExporter } from './clone-exporter.service';
+
+const propertiesToRemove: (keyof GefDeal)[] = [
```

Comment:
I think we should add a test for this - currently, it looks like if a field is removed from here, the tests will pass